assets/constants.js

```diff
@@ -18,5 +18,214 @@ var constants = {
     "Statistics", "Trigonometry", "Welcome"],
   "DISABLED_EXPLORATION_IDS": ["5"],
   "TESTING_CONSTANT": "test",
-  "LIBRARY_TILE_WIDTH_PX": 208
+  "LIBRARY_TILE_WIDTH_PX": 208,
+
+  "//": "Whether to allow custom event reporting to Google Analytics.",
+  "CAN_SEND_ANALYTICS_EVENTS": false,
+
+  "//": "The type of the response returned by a handler when an exception is raised.",
+  "HANDLER_TYPE_HTML": "html",
+  "HANDLER_TYPE_JSON": "json",
+
+  "//": "Prefix for data sent from the server to the client via JSON.",
+  "XSSI_PREFIX": ")]}'\n",
+
+  "DEFAULT_COLOR": "#a33f40",
+  "DEFAULT_THUMBNAIL_ICON": "Lightbulb",
+
+  "//": ["List of supported default categories. For now, each category has a specific ",
+         "color associated with it. Each category also has a thumbnail icon whose ",
+         "filename is '{CategoryName}}.svg'."],
+  "CATEGORIES_TO_COLORS": {
+    "Mathematics": "#cd672b",
+    "Algebra": "#cd672b",
+    "Arithmetic": "#d68453",
+    "Calculus": "#b86330",
+    "Logic": "#d68453",
+    "Combinatorics": "#cf5935",
+    "Graph Theory": "#cf5935",
+    "Probability": "#cf5935",
+    "Statistics": "#cd672b",
+    "Geometry": "#d46949",
+    "Trigonometry": "#d46949",
+
+    "Algorithms": "#d0982a",
+    "Computing": "#bb8b2f",
+    "Programming": "#d9aa53",
+
+    "Astronomy": "#879d6c",
+    "Biology": "#97a766",
+    "Chemistry": "#aab883",
+    "Engineering": "#8b9862",
+    "Environment": "#aba86d",
+    "Medicine": "#97a766",
+    "Physics": "#879d6c",
+
+    "Architecture": "#6e3466",
+    "Art": "#895a83",
+    "Music": "#6a3862",
+    "Philosophy": "#613968",
+    "Poetry": "#7f507f",
+
+    "English": "#193a69",
+    "Languages": "#1b4174",
+    "Latin": "#3d5a89",
+    "Reading": "#193a69",
+    "Spanish": "#405185",
+    "Gaulish": "#1b4174",
+
+    "Business": "#387163",
+    "Economics": "#5d8b7f",
+    "Geography": "#3c6d62",
+    "Government": "#538270",
+    "History": "#3d6b52",
+    "Law": "#538270",
+
+    "Education": "#942e20",
+    "Puzzles": "#a8554a",
+    "Sport": "#893327",
+    "Welcome": "#992a2b"},
+
+  "//": ["List of supported language codes. Each description has a ",
+         "parenthetical part that may be stripped out to give a shorter ",
+         "description."],
+  "ALL_LANGUAGE_CODES": [{
+        "code": "en", "description": "English"
+    }, {
```

Comment:
This should be at the same indentation level as ALL_LANGUAGE_CODES (i.e. 2 spaces not 4); ditto below.